pkg/services/ngalert/api/authorization.go

```diff
@@ -147,6 +147,12 @@ func (api *API) authorize(method, path string) web.Handler {
 			ac.EvalPermission(ac.ActionAlertingProvisioningSetStatus),
 		)
 
+	case http.MethodPost + "/api/convert/{DatasourceUID}/config/v1/rules":
+		eval = ac.EvalAll(
+			ac.EvalPermission(ac.ActionAlertingRuleCreate),
+			// ac.EvalPermission(ac.ActionAlertingRuleExternalRead),
+		)
```

Comment:
I think we must check permissions to read the external rules, i.e.
```
ac.EvalPermission(ac.ActionAlertingRuleExternalRead, datasources.ScopeProvider.GetResourceScopeUID(ac.Parameter(":DatasourceUID")))
```
because under the hood it calls read API.

Also, we may want to check folder read permissions just to fail fast.
```
ac.EvalPermission(dashboards.ActionFoldersRead),
```